src/zope/i18n/__init__.py

```diff
@@ -160,6 +163,9 @@ def translate(msgid, domain=None, mapping=None, context=None,
         domain = msgid.domain
         default = msgid.default
         mapping = msgid.mapping
+        msgid_plural = msgid.msgid_plural
+        default_plural = msgid.default_plural
+        number = msgid.number
```

Comment:
I've been wondering why `Message.__init__()` does

        self.default_plural = default_plural or msgid_plural

when there's no corresponding logic for `self.default`, and when there's no corresponding logic here, if you provide a msgid as a string rather than a Message instance to `translate()`.

From the tests I've seen here I gather that gettext.ngettext() copes fine with a default of `None`?